src/main/java/org/opentripplanner/util/OTPFeature.java

```diff
@@ -33,7 +33,7 @@
     ReportApi(false),
     SandboxAPILegacyGraphQLApi(false),
     SandboxAPIMapboxVectorTilesApi(false),
-    SandboxAPITransmodelApi(false),
+    SandboxAPITransmodelApi(true),
```

Comment:
@Bartosz-Kruba I noticed this was changed. Can you do a PR to revert it. I recommend always using an `otp-config-json`, so that you never need to change the default values.